mystery_o_matic/output/latex/utils.py

```
@@ -0,0 +1,118 @@
+    delimiter = "$$"
+
+def create_tex_template(str):
+    return TexTemplate(str)
+
+def read_tex_template(filename):
+    try:
+        with open(filename, "r", encoding='utf-8') as f:
+            template = create_tex_template(f.read())
+            return template
+    except FileNotFoundError:
+        raise ValueError(f"Template file not found: {filename}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)